spi/common/core-spi/src/main/java/org/eclipse/edc/spi/command/Command.java

```diff
@@ -29,43 +27,5 @@
  * </ul>
  */
 public abstract class Command {
```

Comment:
could this be an `interface`? or rather: is this needed at all? it seems to only have one direct inheritor in production code (`SingleEntityCommand`) and another one in test code

Reply:
yes, I thought the same, I'll remove it and leave the `SingleEntityCommand` as the root class